components/data/data-services/org.wso2.micro.integrator.dataservices.core/src/test/java/org/wso2/micro/integrator/dataservices/core/test/util/TestUtils.java

```diff
@@ -35,7 +35,8 @@
 import javax.xml.validation.SchemaFactory;
 import javax.xml.validation.Validator;
 import java.io.*;
-import java.net.HttpURLConnection;
+import java.net.*;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Using the '.*' form of import should be avoided - java.net.*.](https://app.codacy.com/app/sdkottegoda/micro-integrator/pullRequest?prid=4575789)